src/ontology/edits/oeo-model.omn

```diff
@@ -549,6 +561,134 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1944",
         rdfs:label "has time stamp alignment"@en
     
     
+ObjectProperty: OEO_00390069
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "has iri is a relation between a data item or document and the IRI that identifies it as rescource on the internet.",
+        rdfs:label "has iri"@en
+    
+    Domain: 
+        <http://purl.obolibrary.org/obo/IAO_0000027> or <http://purl.obolibrary.org/obo/IAO_0000310>
+    
+    Range: 
+        OEO_00140178 some rdfs:Literal
```

Comment:
```suggestion
        OEO_00140178 some rdfs:Literal
```
why has number? A literal can also be a string... https://www.w3.org/TR/rdf-schema/#ch_literal

I have the impression that has iri should be a data property itself. Like this, the range could be rdfs:literal directly.